corrects some output in /stats V/v
show flags for own server correct, changed n to N

diff -r 32cd51bef571 ircd/s_stats.c
--- a/ircd/s_stats.c	Sun Jan 11 22:38:38 2009 +0000
+++ b/ircd/s_stats.c	Sun Jan 11 22:38:38 2009 +0000
@@ -513,10 +513,10 @@
                cli_name(cli_serv(acptr)->up),
                IsBurst(acptr) ? 'B' : '-',
                IsBurstAck(acptr) ? 'A' : '-',
-               IsHub(acptr) ? 'H' : '-',
+               (IsHub(acptr) || (IsMe(acptr) && (feature_bool(FEAT_HUB)))) ? 'H' : '-',
                IsService(acptr) ? 'S' : '-',
-               IsIPv6(acptr) ? '6' : '-',
-               IsSendOperName(acptr) ? 'n' : '-',
+               (IsIPv6(acptr) || IsMe(acptr)) ? '6' : '-',
+               (IsSendOperName(acptr) || IsMe(acptr)) ? 'N' : '-',
                cli_hopcount(acptr),
                NumServ(acptr),
                base64toint(cli_yxx(acptr)),
